File: mainn/src/mainn/Employee.java

package mainn;
import java.util.ArrayList;
public class Employee {
    
    
    private String name;
    private String lastName;
    private BankAccount account;
    private ArrayList<BankAccount> accounts ;

    
    public Employee(String name, String lastName) {
        this.name = name;
        this.lastName = lastName;
        this.accounts = new ArrayList<>();
       
    }

    public String getName() {
        return name;
    }

    public String getLastName() {
        return lastName;
    }

    public BankAccount getAccount() {
        return account;
    }
    
    public void setAccount(BankAccount account) {
        this.account = account;
    }

   
    
    

    public void almacenarCuentas( BankAccount accountNumber )
     {
         accounts.add(accountNumber);
         
     }
    
     public ArrayList<BankAccount> getAccounts() {
        return accounts;
    }
    
    
    public void getData()
    {
        System.out.println(String.format("Nombre: %s",  name));
        //System.out.println(String.format("Total de cuentas: %d",accounts.size()));
        System.out.println(String.format("Total de cuentas: %d",accounts.size()));
        for (int i = 0; i < accounts.size(); i++)
        {
            
            
            System.out.println(String.format("Numero de cuenta: %d", getAccounts().get(i).getAccountNumber()));
            System.out.println(String.format("Saldo de la cuenta: %f", getAccounts().get(i).getAmount()));
            System.out.println(String.format("Tipo de cuenta %s",  getAccounts().get(i).getType()));
            System.out.println("");
        }        
    }
}
    

File: employee.py
class Employee:
     __name = ""
     __last_name = ""
     __accounts = []
     
     def __init__(self, name,  last_name,  accounts) :
     
        self.__name = name
        self.__last_name = last_name
        self.__accounts = accounts
        
     def  get_name(self) :
        return self.__name


     def get_last_Name(self) :
        return self.__last_name
     
     def get_accounts(self):
        return self.__accounts
    
     def get_Data(self):
         print("Nombre: "+self.__name)
         print("Total de cuentas: {}".format(len(self.__accounts)))
         for account in self.__accounts:
            print("  - Número de cuenta: {}".format(account.get_account_number()))
            print("    Saldo de la cuenta: {}".format(account.get_amount()))
            print("    Tipo de cuenta: {}".format(account.get_type()))
